os/drivers/audio/alc5658.c

```diff
@@ -974,8 +974,31 @@ static int alc5658_ioctl(FAR struct audio_lowerhalf_s *dev, int cmd, unsigned lo
 	switch (cmd) {
 
 	case AUDIOIOC_PREPARE: {
-		/* TODO: Need to be implemented later */
-		audvdbg("AUDIOIOC_PREPARE: No Action Would be taken now \n");
+		/* This is to aid Tinyalsa pcm prepare, this would stop dma, i2s and
```

Comment:
You'd better remove comment regarding Tinyalsa, AUDIOIOC_PREPARE can be sent by any layer where using ioctl.